TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -8265,8 +8265,11 @@ selectStatement: fullselect;
 fullselect: subselect;
 subselect: sql_selectClause;
 sql_selectClause: 
-SQL_SELECT (SQL_ALL|SQL_DISTINCT)? star;
-
+  SQL_SELECT (SQL_ALL|SQL_DISTINCT)? (star | selections) ;
+selections: selection (SQL_CommaSeparator selection)*;
+selection: dotStarSelection;
+dotStarSelection: tableOrViewOrCorrelationName  dot star; 
+tableOrViewOrCorrelationName : ((DBMS=UserDefinedWord dot)? (SchemaName=UserDefinedWord dot))? (Name=UserDefinedWord); 
```

Comment:
Can you add links to DB2 documentation in order to see the syntax rules defined by IBM ?
At least for `sql_selectClause` and `tableOrViewOrCorrelationName`.